exporter/azuredataexplorerexporter/metricsdata_to_adx.go

```diff
@@ -248,9 +249,7 @@ func rawMetricsToAdxMetrics(_ context.Context, metrics pmetric.Metrics, logger *
 }
 
 func copyMap(toAttrib, fromAttrib map[string]any) map[string]any {
```

Comment:
I realize you're probably trying to keep this as minimal as possible, but this method is no longer needed. Could we just replace all references with `maps.Copy`?